server/authentication/refreshToken.ts

```diff
@@ -59,15 +59,25 @@ export const checkTokenValidityAndUpdate = async (req: Request, res: Response, n
 
   const { idToken, refreshToken } = req.user
 
-  if (tokenIsValid(idToken, nowMinus5Minutes(Date.now()))) {
+  if (
+    tokenIsValid(
```

Comment:
@tosin-ogunrinde I think this function `tokenIsValid` is wrong, the condition needs to be the other way around?